code401challenges/src/main/java/GraphExercises/Graph.java

```diff
@@ -0,0 +1,77 @@
+package GraphExercises;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+public class Graph<T>{
+
+    public HashMap<Vertex<T>, List<Edge<T>>> adjacencyList;
+    public int size = 0;
+
+//  Constructor
+    public Graph() {
+        this.adjacencyList = new HashMap<>();
+    }
+
+//  Getters & Setters
+
+    public HashMap<Vertex<T>, List<Edge<T>>> getAdjacencyList() {
+        return adjacencyList;
+    }
+
+    public void setAdjacencyList(HashMap<Vertex<T>, List<Edge<T>>> adjacencyList) {
```

Comment:
If you're going to set the AdjacencyList, then you should also update the size since size of the list passed could be different.